tools/telemetry-generator/src/handlers/bundleSizeHandler.ts

```diff
@@ -0,0 +1,26 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = function handler(fileData, logger) {
+    // - fileData is a JSON object obtained by calling JSON.parse() on the contents of a file.
+    // - logger is an ITelemetryBufferedLogger. Call its send() method to write the output telemetry
+    //   events.
+        if (fileData.assets?.length === undefined || fileData.assets?.length === 0) {
+            console.log(`could not locate assets info`);
+        }
+
+        for (const asset of fileData.assets) {
+            // we only need .js files
+            if (asset.size >= 0 && asset.name !== undefined && asset.name.toLowerCase().indexOf(".js", asset.name.length - ".js".length) !== -1) {
```

Comment:
I think you can just check .endsWith instead of ".indexOf(".js", asset.name.length - ".js".length) !== -1"